bounties/level-2/stellar-accounts-as-a-service.md

```diff
@@ -0,0 +1,190 @@
+This approach is error-prone. Users can forget to add the memo, or they can input the wrong memo.
+
+To solve this issue [muxed accounts](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0027.md) were added to Stellar. Muxed accounts are essentially a way to combine Stellar addresses with integer identifiers.
+Payments sent to a muxed account are actually sent to the account used to create the muxed account. However, the muxed account identifier is also added to the transaction.
+
+For this bounty the task is to create an API for a [custodial](https://www.gemini.com/cryptopedia/crypto-wallets-custodial-vs-noncustodial) wallet service using muxed accounts.
+A simple database should be used to keep track of the registered users, their muxed account addresses and their balances.
+
+The api should have the following endpoints:
+
+```
+openapi: 3.0.1
+info:
+  title: Stellar accounts as a service
+  description: 'This is the api specification for the Stellar accounts as a service bounty.'
+  version: 1.0.0
+tags:
+- name: auth
+  description: Authenticate yourself
+- name: info
+  description: Info
+- name: pay
+  description: Different ways to pay
+paths:
+  /register:
+    post:
+      tags:
+      - auth
+      summary: Create an account.
+      requestBody:
+        description: Credentials to register with.
+        content:
+          application/json:
+            schema:
+              properties:
+                username:
+                  type: string
+                password:
+                  type: string
+        required: true
+      responses:
+        400:
+          description: Invalid body.
+        500:
+          description: Catch all response for other errors aside from the ones listed.
+          content:
+            application/json:
+              schema:
+                properties:
+                  error:
+                    type: string
+                    description: Text explaining the error.
+        
+  /login:
+    post:
+      tags:
+      - auth
+      summary: Get an auth token.
+      requestBody:
+        description: Credentials to login with.
+        content:
+         application/json:
+          schema:
+            properties:
+              username:
+                type: string
+              password:
+                type: string
+        required: true
+      responses:
+        200:
+          description: Successful login.
+          content:
+            application/json:
+              schema:
+                properties:
+                  apikey:
+                    type: string
+        400:
+          description: Invalid Body.
+  /info:
+    get:
+      tags:
+      - info
+      security:
+        - bearerAuth : []
+      summary: Get info about the user's address and XLM balance.
+      responses:
+        200:
+          description: Sucessful operation.
+          content:
+            application/json:
```

Comment:
You can add examples here. Either for the whole schema or per property. No must, just a suggestion ;)